src/homeWork10/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,31 @@
+package homeWork10;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class FileLoggerConfigurationLoader {
+
+    public static FileLoggerConfiguration load() {
+
+        String fileName;
+        String fileFormat;
+        LoggingLevel loggingLevels;
+        int fileMaxSize;
+
+        try (InputStream is = new FileInputStream("config.properties")) {
```

Comment:
Remove `(InputStream is = new FileInputStream("config.properties"))` as it is redundant and you're using `Properties` to load our configuration, which is correct.